src/components/VFooter/VFooter.vue

```diff
@@ -99,66 +101,99 @@ export default defineComponent({
   setup(props) {
     const { all: allPages, current: currentPage } = usePages(true)
 
-    const isContentMode = computed(() => {
-      if (props.mode) {
-        return props.mode === 'content'
-      } else {
-        return ['search', 'audio', 'image'].some((prefix) =>
-          currentPage.value?.startsWith(prefix)
-        )
-      }
-    })
+    const isContentMode = computed(() => props.mode === 'content')
```

Comment:
The wrapper component should know whether the footer is in 'content' mode or in the 'internal' mode.